en/lessons/gravity-model.md

```diff
@@ -0,0 +1,712 @@
+| Cumberland | 13 | 407.3 | 96862 | 78 | 38.46 | 64.42 |
+| Derbyshire | 28 | 196.9 | 122593 | 75 | 48.00 | 68.02 |
+| Devon | 98 | 272.5 | 279652 | 89 | -7.87 | 69.98 |
+| Dorset	| 27 | 176.8 | 97262 | 81 | 22.22 | 67.30 |
+| Durham | 25 | 380.7 | 119779 | 78 | 33.33 | 63.16 |
+| Gloucestershire | 162 | 157.1 | 215576 | 81 | 	9.88 | 66.54 |
+| Hampshire | 78 | 102.4 | 166648 | 96 | 6.25 | 61.45|
+| Herefordshire | 45 | 190.5 | 81882 | 70 | 28.57 | 62.05 |
+| Hertfordshire | 99 | 35.3 | 95868 | 90 | 4.44 | 63.82 |
+| Huntingdonshire | 21 | 87.5 | 35370 | 89 | 7.87 | 58.72 |
+| Lancashire | 94 | 281.8 | 301407 | 78 | 55.13 | 71.65 |
+| Leicestershire | 20 | 146.1 | 107028 | 79 | 65.82 | 64.84 |
+| Lincolnshire | 41 | 179.8 | 181814 | 84 | 26.19 | 58.73 |
+| Northamptonshire | 33 | 107.6 | 128798 | 78 | 21.79 | 63.81 |
+| Northumberland | 58 | 440.0 | 148148 | 72 | 70.83 | 58.22 |
+| Nottinghamshire | 31 | 187.5 | 98216 | 108 | 0.00 | 61.30|
+| Oxfordshire | 78 | 86.8 | 99354 | 84 | 25.00 | 64.23 |
+| Rutland | 2 | 132.5 | 15123 | 90 | 10.00 | 64.12 |
+| Shropshire | 75 | 214.0 | 147303 | 76 | 18.42 | 66.50 |
+| Somerset | 159 | 180.4 | 234179 | 77 | 3.90 | 68.29 |
+| Staffordshire | 82 | 185.3 | 175075 | 76 | 18.42 | 67.80 |
+| Warwickshire | 104 | 149.3 | 152050 | 96 | -3.13 | 65.05 |
+| Westmorland | 5 | 365.0 | 38342 | 74 | 62.16 | 71.05|
+| Wiltshire | 99 | 131.7 | 182421 | 84 | 20.24 | 63.64 |
+| Worcestershire | 94 | 164.4 | 130757 | 81 | 25.93 | 65.78 |
+| Yorkshire | 127 | 282.2 | 651709 | 80	 | 58.33 | 61.87 |
+
+The final difference between this formula and the one used in the original article, is that two of the variables happen to have a stronger relationship with vagrancy when plotted naturally logarithmically. They are population at origin ($$P$$) and distance from origin to London ($$d$$). What this means is that for the data in this study, the regression line (sometimes called line of best fit) is a better fit when the data has been logged than when it has not been. You can see this in Figure 7, with the non-logged population figures on the left, and the logged version on the right. More of the points are closer to the line of best fit on the logged graph than on the non-logged one.
+
+{% include figure.html filename="figure7.png" caption="Figure 7: Number of Vagrants plotted against population at origin (left), and natural log of population of origin (right) with a simple regression line overlayed on both. Note the stronger relationship between the two variables visible on the second graph." %}
+
+Because this is the case with this *particular* data (your own data in a similar type of study may not follow this pattern), the formula was adjusted to use the naturally logged versions of these two variables, resulting in the FINAL formula used in the gravity model (Figure 8). We could not possibly have known about the need for this adjustment until after we had collected our variable data:
```

Comment:
Use italics or bold to emphasize a term, not capitalization

Reply:
I've removed this unnecessary capitalization.